packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -333,6 +333,24 @@ export const components = ({
         }),
       },
     },
+    MuiBadge: {
+      styleOverrides: {
+        badge: ({ ownerState }) => ({
+          ...(ownerState.color === "primary" && {
+            backgroundColor: odysseyTokens.HueBlue500,
```

Comment:
Whenever possible, use the semantic rather than palette token (ie, `PalettePrimaryMain` rather than `HueBlue500`. These aren't always available, so fall back to the `Hue` versions when needed.